app/assets/javascripts/discourse/tests/acceptance/sidebar-user-community-section-test.js

```diff
@@ -1134,6 +1127,47 @@ acceptance("Sidebar - Logged on user - Community Section", function (needs) {
 
     assert.ok(teardownCalled, "section link teardown callback was called");
   });
+
+  test("pressing esc closes the more menu", async function (assert) {
+    await visit("/");
+
+    await click(
+      ".sidebar-section[data-section-name='community'] .sidebar-more-section-links-details-summary"
+    );
+
+    assert.ok(exists(".sidebar-more-section-links-details-content"));
+
+    const event = new KeyboardEvent("keydown", { key: "Escape" });
+    document.dispatchEvent(event);
```

Comment:
```suggestion
    await triggerKeyEvent(document, "keydown", "Escape");
```

(you'll need to import `triggerKeyEvent` from `@ember/test-helpers`)